eng/pipelines/common/global-build-job.yml

```diff
@@ -136,15 +137,15 @@ jobs:
           artifact: Mono_Offsets_${{monoCrossAOTTargetOS}}
           path: '$(Build.SourcesDirectory)/artifacts/obj/mono/offsetfiles'
 
-    - ${{ if eq(parameters.buildingOnSourceBuildImage, true) }}:
+    - ${{ if eq(parameters.isSourceBuild, true) }}:
       - template: /eng/common/templates/steps/source-build.yml
         parameters:
           platform:
             buildScript: $(_sclEnableCommand) $(Build.SourcesDirectory)$(dir)build$(scriptExt)
-            nonPortable: true
-            # Use a custom RID that isn't in the RID graph here to validate we don't break the usage of custom rids that aren't in the graph.
-            targetRID: banana.24-x64
```

Comment:
You remove the custom RID, but we want to keep it to make sure that we can build with arbitrary RIDs.

Reply:
Then we probably need another leg, or one that only runs in PRs? Preference?

Reply:
@ViktorHofer @tmds @MichaelSimons I made some tweaks:

- Build only linux-x64 portable officially (nothing else flows)
- Add missing PR legs for linux-x64 (to avoid official breaks)
- Add rolling CI for Banana and centos7